appinventor/appengine/src/com/google/appinventor/client/explorer/commands/CopyYoungAndroidProjectCommand.java

```diff
@@ -140,7 +141,21 @@ public CopyProjectDialog(final ProjectRootNode oldProjectNode) {
       }
 
       newNameTextBox = new LabeledTextBox(checkpoint ? MESSAGES.checkpointNameLabel()
-          : MESSAGES.newNameLabel());
+          : MESSAGES.newNameLabel(), new Validator(){
+        @Override
+        public boolean validate(String value) {
+          errorMessage = TextValidators.getErrorMessage(value);
+          if (errorMessage.length()>0){
```

Comment:
you could basically `return !errorMessage.length() > 0;`
or `return errorMessage.length < = 0;` instead of using the negation.